lte/gateway/deploy/roles/ghz/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+ghz_root: /var/tmp/ghz
```

Comment:
not sure if /var/tmp is the best place for this

Reply:
+1 

Use `/tmp/ghz`